### PR TITLE
[SCB-1663] add scripts

### PR DESCRIPTION
**Pull request by kakulisen**
I wrote two scripts for the cli module
One is a shell script for Linux and the other is a batch script for windows
At the same time, I updated the documentation

But I don't know if these scripts are named properly and if it runs successfully on your machine
Please help to check

### CODE REVIEW

**Comment by chanjarster** [comment]
I build the release binary and I checked on Mac and got this:

```txt
readlink: illegal option -- f
usage: readlink [-n] [file ...]
usage: dirname path
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
error: cli.jar not exist. make sure it is placed in the current directory or subdirectory of this script
```

Seems like an issue when you trying to got the script's directory. Check [this](https://github.com/apache/tomcat/blob/master/bin/startup.sh#L28-L41) out, maybe will help you.

**Comment by kakulisen** [comment]
@chanjarster I tried the method you recommended. And tested successfully on my friend's mac. I have submitted this change. You can re-verify it

File: cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
 
     initialProjectVersion();
 
-    Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("java -jar toolkit-cli-" + projectVersion + ".jar");
+    Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("cli.sh");
```
Comment (chanjarster) [inline cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java]: Not good, it's bind to linux

**Comment by chanjarster** [comment]
@kakulisen yeah, it works. I think you can add a stage in travis ci to test if this script is ok, use this command `cli.sh help` is just fine.

File: cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java (hunk 1 above)
Comment (kakulisen) [reply cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java]: Yes, it's not good. I will try to fix it

Comment (kakulisen) [reply cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java]: I use system properties -Dscript.name to decide the name of the script
What do you think about it?

File: cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java [hunk 2]
```diff
@@ -33,7 +35,14 @@ public static void main(String[] args) {
...
     initialProjectVersion();
 
-    Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("java -jar toolkit-cli-" + projectVersion + ".jar");
+    String scriptName = System.getProperty("script.name");
+    Cli.CliBuilder<Runnable> builder = null;
+    if (StringUtils.isNotEmpty(scriptName)) {
+      builder = Cli.builder(scriptName);
+    } else {
+      builder = Cli.builder("java -jar toolkit-cli-" + projectVersion + ".jar");
+    }
```
Comment (chanjarster) [inline cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java]: @kakulisen Yeah, it's better now. One more thing, the executable jar actually built is named 'cli-{version}.jar' not 'toolkit-cli-{version}.jar'

File: cli/scripts/cli.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```
Comment (chanjarster) [inline cli/scripts/cli.sh]: @kakulisen yeah, it works. I think you can add a stage in travis ci to test if this script is ok, use this command cli.sh help is just fine.

File: cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java (hunk 2 above)
Comment (chanjarster) [inline cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java]: @kakulisen Yeah, it's better now. One more thing, the executable jar actually built is named 'cli-{version}.jar' not 'toolkit-cli-{version}.jar'

**Comment by kakulisen** [comment]
@chanjarster 
According to your suggestions, I add a stage in travis ci to test cli.sh

And I try to find a way to run tests in windows in travis ci  and encountered many problems
such as `The language 'java' is currently unsupported on the Windows Build Environment.`

I tried the solution linked below
`https://travis-ci.community/t/java-support-on-windows/286`
But it creates more problems. This may take a long time to resolve it

File: cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java (hunk 2 above)
Comment (kakulisen) [reply cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java]: I have fixed it

**Comment by chanjarster** [comment]
@kakulisen Yeah, according to travis [doc](https://docs.travis-ci.com/user/reference/windows/#supported-languages), currently java on windows is not supported yet. We can leave this issue later.

**Pull request merged**